app/src/main/res/values-ar/strings.xml

```diff
@@ -208,6 +208,10 @@
     <string name="allow_changing_time_zones">السماح بتغيير المناطق الزمنية للحدث</string>
     <string name="manage_quick_filter_event_types">إدارة أنواع التصفية السريعة</string>
     <string name="allow_creating_tasks">السماح بإنشاء المهام</string>
+    <string name="backups">Backups</string>
+    <string name="enable_automatic_backups">Enable automatic backups</string>
+    <string name="manage_automatic_backups">Manage automatic backups</string>
+    <string name="date_time_pattern_info">You can use the following pattern to name your file automatically:\n\n%Y - year\n%M - month\n%D - day\n%h - hour\n%m - minute\n%s - second</string>
```

Comment:
guess it should be "patterns" instead of "pattern"

Reply:
right, fixed it